resources.whatwg.org/build/deploy.sh

```diff
@@ -78,20 +77,17 @@ run_post_build_step() {
     fi
 }
 
-if [[ $BRANCH != "master" ]] ; then
-    # Branch snapshot, if not master
-    BRANCH_DIR="$WEB_ROOT/$BRANCHES_DIR/$BRANCH"
-    mkdir -p "$BRANCH_DIR"
-    curl https://api.csswg.org/bikeshed/ -f -F file=@"$INPUT_FILE" -F md-status=LS-BRANCH \
-         -F md-warning="Branch $BRANCH $BRANCH_URL_BASE$BRANCH replaced by $LS_URL" \
-         -F md-title="$TITLE (Branch Snapshot $BRANCH)" \
-         -F md-Text-Macro="SNAPSHOT-LINK $BACK_TO_LS_LINK" \
-         > "$BRANCH_DIR/index.html";
-    copy_extra_files "$BRANCH_DIR"
-    run_post_build_step "$BRANCH_DIR"
-    echo "Branch snapshot output to $BRANCH_DIR"
-else
-    # Commit snapshot, if master
+# Living standard
```

Comment:
This used to only run in the if $BRANCH is "master". Seems better to keep it that way?

Reply:
That seemed broken to me, then we will effectively do nothing useful and should instead exist early above. And doing the build and the validator steps while the PR is still open seems better.

Reply:
But don't we then update the website with the contents of the PR?

Reply:
No, see the change after "Deploy from Travis on master branch only"

Reply:
Ah I see. I think this looks good then, but I'll let @domenic confirm.

Reply:
Note that we're already, per line 33, skipping all builds for a pull request. Was your intention to change that? If not, this doesn't make too much sense.

Reply:
AFAICT, that check is there to skip what we consider redundant work, seen working here:
https://travis-ci.org/whatwg/fetch/builds/309802756
https://travis-ci.org/whatwg/fetch/builds/309802707

Note that we can also do that by disabling one of "Build branch updates" or "Build pull request updates" in Travis.

I'd like to make some changes here, but then the drive-by changes eclipse the original ones, so I'll simplify this and do a follow up.